src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -285,11 +286,11 @@ public static void deleteTempSourceBQTable() throws IOException, InterruptedExce
   @Before(order = 1, value = "@BQ_PARTITIONED_SOURCE_TEST")
   public static void createTempPartitionedSourceBQTable() throws IOException, InterruptedException {
     bqSourceTable = "E2E_SOURCE_" + UUID.randomUUID().toString().replaceAll("-", "_");
-    BigQueryClient.getSoleQueryResult("create table `test_automation." + bqSourceTable + "` " +
+    BigQueryClient.getSoleQueryResult("create table `bq_automation." + bqSourceTable + "` " +
```

Comment:
similar comment at all places using the hard-coded value.

Reply:
Resolved.